src/Actions/Player/Finger.cs

```diff
@@ -57,52 +72,19 @@ public override void Execute(ActionInput actionInput)
-                sb.AppendLine("Description: " + target.Description);
-                sb.AppendLine("Full Name: " + target.FullName);
-                sb.AppendLine("ID: " + target.Id);
-                sb.AppendLine("Type: " + target.GetType().Name); // identify npc ?
-                sb.AppendLine("Race: TBA      Guild: TBA     Class: TBA");
-                sb.AppendLine("Religion: TBA");
-                sb.AppendLine("Gender: TBA");
-                sb.AppendLine("Email: TBA         Messenger Id: TBA");
-                sb.AppendLine("Home Page: TBA");
-                sb.AppendLine("Spouse: TBA");
-                sb.AppendLine("Creation Date: TBA");
-                sb.AppendLine("Age: TBA");
-
-                if (isOnline)
-                {
-                    sb.AppendLine("Status: <%green%>Online<%n%>"); // need way to report both offline and online
-                    sb.AppendLine("Location: " + target.Parent.Name);
-                    sb.AppendLine(string.Format("Last Login: {0}", playerBehavior.PlayerData.LastLogin));
-                    sb.AppendLine(string.Format("Current IP Address: {0}", addressIP));
-                }
-                else
-                {
-                    sb.AppendLine("Status: <%red%>Offline<%n%>"); // need way to report both offline and online
-                    sb.AppendLine(string.Format("Last Login: {0}", playerBehavior.PlayerData.LastLogin));
-                }
-
-                sb.AppendLine("Plan: TBA");
-                sb.AppendLine("MXP: TBA");
-            }
-            else
-            {
-                sb.AppendLine("<%yellow%><%b%>Name: " + target.Name + " Title: " + target.Title + "<%n%>");
-                sb.AppendLine("Description: " + target.Description);
-                sb.AppendLine("Full Name: " + target.FullName);
-                sb.AppendLine("ID: " + target.Id);
-                sb.AppendLine("Type: " + target.GetType().Name); // identify npc ?
-                sb.AppendLine("Race: TBA      Guild: TBA     Class: TBA");
-                sb.AppendLine("Religion: TBA");
-                sb.AppendLine("Gender: TBA");
-                sb.AppendLine("Email: TBA         Messenger Id: TBA");
-                sb.AppendLine("Home Page: TBA");
-                sb.AppendLine("Spouse: TBA");
-                sb.AppendLine("Creation Date: TBA");
-                sb.AppendLine("Age: TBA");
+                
+                string statusString = isOnline ? "<%green%>Online<%n%>" : "<%red%>Offline<%n%>";
+                
+                sb.AppendAnsiLine($"Status: {statusString}"); // need way to report both offline and online
+                sb.AppendAnsiLine($"Location: {target.Parent.Name}");
+                sb.AppendAnsiLine($"Last Login: {playerBehavior.PlayerData.LastLogin}");
+                
+                if(isOnline)
+                    sb.AppendAnsiLine($"Current IP Address: {addressIP}");
```

Comment:
Ah, while this command is hardly restricted (any Player can use it), this is a good example of additional info that should probably only be presented to an admin.

If you'd like to try improving the conditional to also only show this part to a command sender of sufficient security role, you are welcome to.